mmv1/products/deploymentmanager/product.yaml

```diff
@@ -17,5 +17,6 @@ display_name: 'Cloud Deployment Manager'
 versions:
   - name: 'ga'
     base_url: 'https://www.googleapis.com/deploymentmanager/v2/'
+    cai_base_url: 'https://deploymentmanager.googleapis.com/v2/'
```

Comment:
Oh actually one thing - I _think_ the CAI base url shouldn't contain the version.

Reply:
see https://cloud.google.com/asset-inventory/docs/supported-asset-types

Reply:
True, but all of the other instances of `cai_base_url` in the repo appear to include the version. We parse it for the metadata generation, and looks like we do the same for TGC, so I think we want to keep it.

Reply:
SGTM - we can fix later